### PR TITLE
[WIP] Upgrade redux

### PR DESCRIPTION
**Pull request by thangngoc89**
Ref: #60

### CODE REVIEW

File: src/client/index.js [hunk 1]
```diff
@@ -45,10 +37,12 @@ export default function statinamic({
     <div id="statinamic-container">
       <MetadataProvider metadata={ metadata }>
         <Provider store={ store }>
-          <Router history={ history } routes={ routes } />
+          <div>
```
Comment (MoOx) [inline src/client/index.js]: Not a fan about the extra div.
FYI, the server side is here: https://github.com/MoOx/statinamic/blob/cb000b78d0172e05dadffeb8bdb0fecbe770/src/static/to-html/url-as-html.js

**Comment by MoOx** [comment]
Can you rebase? Master should be good now. Also please consider my note about the div.

File: src/client/index.js (hunk 1 above)
Comment (thangngoc89) [inline src/client/index.js]: @MoOx 
How would you avoid extra div ? 

devtools component needs to be inside Provider
and Provider only accepts a child

Comment (MoOx) [inline src/client/index.js]: Oh yes, indeed. so can you adjust the static generation part?

Comment (thangngoc89) [inline src/client/index.js]: okie. Pushed some code.
waiting for Travis

**Comment by thangngoc89** [comment]
@MoOx rebased, ready to be merged

**Comment by MoOx** [comment]
Thanks!

**Pull request merged**